atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/BackgroundSweeperImpl.java

```diff
@@ -94,13 +95,26 @@ public BackgroundSweeperImpl(LockAwareTransactionManager txManager,
                                  Supplier<Long> sweepPauseMillis,
                                  Supplier<Integer> sweepBatchSize,
                                  SweepTableFactory tableFactory) {
+        this(txManager, kvs, sweepRunner, isSweepEnabled, sweepPauseMillis, sweepBatchSize, tableFactory,
+                Optional.absent());
+    }
+
+    public BackgroundSweeperImpl(LockAwareTransactionManager txManager,
```

Comment:
We tend to accept API breaks (we're pre-1.0). If it's difficult to propagate the change in the internal product immediately, another option could be to use the new method in AtlasDB and `@Deprecate` the old one.